theme.json

```diff
@@ -88,11 +88,21 @@
           "radius": false
         }
       },
+      "core/heading": {
```

Comment:
Also needed by https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/914 but of course by the rest of blocks / patterns.

Reply:
I don't see how it's needed in the PR that you referenced here 🤔 And I'm not sure we have other patterns that would require this!

Reply:
Removed that and keep only the feature to `core/heading`